nixos/modules/config/nsswitch.nix

```diff
@@ -15,6 +15,7 @@ let
   ldap = canLoadExternalModules && (config.users.ldap.enable && config.users.ldap.nsswitch);
   sssd = canLoadExternalModules && config.services.sssd.enable;
   resolved = canLoadExternalModules && config.services.resolved.enable;
+  googleOSLogin = canLoadExternalModules && config.security.googleOSLogin.enable;
```

Comment:
I'm not sure on why some modules seem to set this here, and some from their module itself. Any pointers?

Reply:
No idea why. In the case of googleOSLogin it would be better if this could be set by the module itself as it's very target-specific.

Reply:
The main reason I chose to set this here is that `hostArray` cant be modified externally right now.
If we added facilities for that I'd like it if each individual module took care of it's own `nss` config.

Reply:
@flokli You meant `passwdArray`, and in this case, it's like all other modules are currently doing it too. In some cases, order matters here, so it makes sense to do it inside the nsswitch module.